editor/code_editor.cpp

```diff
@@ -391,10 +410,10 @@ void FindReplaceBar::_update_matches_label() {
 
 		if (results_count == 0) {
 			matches_label->set_text("No match");
-		} else if (results_count == 1) {
-			matches_label->set_text(vformat(TTR("%d match"), results_count));
+		} else if (results_count_to_current == -1) {
+			matches_label->set_text(vformat(results_count == 1 ? TTR("%d match") : TTR("%d matches"), results_count));
```

Comment:
I think this can be replaced with plural TTRN.

Reply:
Thanks for pointing out.